pkg/lsync/lfrequentaccess_test.go

```diff
@@ -0,0 +1,143 @@
+/*
+ * Minio Cloud Storage, (C) 2017-2020 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// GOMAXPROCS=10 go test
```

Comment:
```suggestion
// go test -cpu=10
```
Not entirely sure what you are trying to tell here ;)

Reply:
These are all old number https://github.com/minio/lsync from here, the project was implemented to implement timeouts for lockers. 

I am not changing anything or adding anything new here @klauspost